engine/common/public_primary_keys.py

```diff
@@ -7,48 +7,13 @@
 logger = logging.getLogger(__name__)
 
 
-def generate_public_primary_key(prefix, length=settings.PUBLIC_PRIMARY_KEY_MIN_LENGTH):
-    """It generates random string with prefix and length
-    :param prefix:
-            "U": ("user_management", "User"),
-            "O": ("user_management", "Organization"),
-            "T": ("user_management", "Team"),
-            "N": ("base", "UserNotificationPolicy"),
-            "C": ("alerts", "AlertReceiveChannel"),
-            "R": ("alerts", "ChannelFilter"),
-            "S": ("schedules", "OnCallSchedule"),
-            "E": ("alerts", "EscalationPolicy"),
-            "F": ("alerts", "EscalationChain"),
-            "I": ("alerts", "AlertGroup"),
-            "A": ("alerts", "Alert"),
-            "M": ("alerts", "ResolutionNote"),
-            "G": ("slack", "SlackUserGroup"),
-            "K": ("alerts", "CustomButton"),
-            "O": ("schedules", "CustomOnCallShift"),
-            "B": ("heartbeat", "IntegrationHeartBeat"),
-            "H": ("slack", "SlackChannel"),
-            "Z": ("telegram", "TelegramToOrganizationConnector"),
-            "L": ("base", "LiveSetting"),
-            "X": ("extensions", "Other models from extensions apps"),
```

Comment:
I think it's easy for this comment to go out of date. I know for example webhooks and shift swap requests are not listed here. I think it makes more sense to remove.